src/structs/Scale.jl

```diff
@@ -0,0 +1,11 @@
+struct Scale
+    x::Float64
+    y::Float64
+end
+
+Base.:+(s1::Scale, s2::Scale) = Scale(s1.x + s2.x, s1.y + s2.y)
+Base.:-(s1::Scale, s2::Scale) = Scale(s1.x - s2.x, s1.y - s2.y)
+Base.:*(s1::Scale, f::Number) = f * s1
+Base.:*(f::Number, s1::Scale) = Scale(f * s1.x, f * s1.y)
+Base.:*(s1::Scale, s2::Scale) = Scale(s1.x * s2.x, s1.y * s2.y)
+Base.:/(s1::Scale, s2::Scale) = Scale(s1.x / s2.x, s1.y / s2.y)
```

Comment:
Curious about this - wasn't it discussed on the Julia Slack that dispatching these base Julia definitions is a sort of an anti-pattern? Could you explain why you are providing this syntactic ability?

Reply:
Can you link me to that discussion if it still exists? :smile: 
I think it makes sense to define `+` and `-` and `*` such that interpolation works. It's not type piracy as we own `Scale` so I think it should be fine?

Reply:
This was the part of the discussion I was referencing off the Slack (wasn't lost yet!):

![image](https://user-images.githubusercontent.com/29561456/97326111-0b3b1500-184a-11eb-8dcc-64d3b3bf028e.png)

I agree with your logic behind it not being type piracy. I just wanted to make sure we aren't introducing bad practices into Javis via this approach. Feel free to resolve this issue if you feel things are not a problem.

Reply:
Yeah I think `Object + Action` would fall into that category of syntactic sugar but this is like
```julia-repl
julia> using Luxor
[ Info: Precompiling Luxor [ae8d54c2-7ccd-5906-9d76-62fc9837b5bc]

julia> Point(1,2)*2
Point(2.0, 4.0)
```